products/bash/ar_remote_create.sh

```diff
@@ -139,14 +144,6 @@ function test_remote() {
 }
 
 ## retries request until we get a 200, or hit max retries
-echo "ar_path=$ar_path"
-echo "token=$token"
-echo "remote=$remote"
-
-
-curl -X POST --insecure -v https://$ar_path/api/remotes/test -d "$remote" -H "Content-Type: application/json" -H "Authorization: Bearer $token" -o ./ar_create_tmp/remote-status.log 
```

Comment:
Why has this stuff been gotten rid of sorry? It looks like it's a verification that the remote has been setup?

Reply:
That's debug code I added in, somehow got on to the operators branch, that's not on master.